assignments/4. operators/conditional-operator.md

```diff
@@ -44,6 +94,39 @@ Take a number value from user and alert the message if it matches the conditions
 * [ ] PLEASE TRY AGAIN, if  is none of the above.
 ```js
 // Your code goes here
+var number = Number(prompt('Enter any number'));
+switch(number) {
+  case 1 :
+  ONE;
+  break;
+   case 2 :
+  TWO;
+  break;
+   case 3 :
+  THREE;
+  break;
+   case 4 :
+  FOUR;
+  break;
+   case 5 :
+  FIVE;
+  break;
+   case 6 :
+  SIX;
+  break;
+   case 7 :
+  SEVEN;
+  break;
+   case 8 :
+  EIGHT;
+  break;
+   case 9 :
+  NINE;
+  break;
+  default:
+  PLEASE TRY AGAIN;
+}
+
```

Comment:
You used `ONE` `TWO` etc it's not string ("") so it will be treated a variable. Fix it.